packages/yasqe/grammar/sparql11-grammar.pl

```diff
@@ -356,9 +357,10 @@
 	[object,*([',',object])].
 %[80]
 object ==>
-	[graphNode].
+	[graphNode, ?(annotationPattern)].
 %[81]
-triplesSameSubjectPath ==> [varOrTerm,propertyListPathNotEmpty].
+triplesSameSubjectPath ==> [varOrTermOrQuotedTP,propertyListPathNotEmpty].
+% TODO: triplesNodePath[100] or triplesNode[98] below?
```

Comment:
This one is different than the docs, but I did not change as it's not directly related to the change here. Leaving a TODO for later™.